src/api/tesseractmain.cpp

```diff
@@ -299,12 +299,11 @@ static void SetVariablesFromCLArgs(tesseract::TessBaseAPI* api, int argc,
 }
 
 static void PrintLangsList(tesseract::TessBaseAPI* api) {
-  GenericVector<STRING> languages;
+  std::vector<std::string> languages;
   api->GetAvailableLanguagesAsVector(&languages);
   printf("List of available languages (%d):\n", languages.size());
```

Comment:
```suggestion
  printf("List of available languages (%zu):\n", languages.size());
```

Reply:
This should fix the issue reported by LGTM. I don't add that now to allow the continuous integration to finish.